tools/system_libs.py

```diff
@@ -750,10 +750,10 @@ class crt1(MuslInternalLibrary):
   name = 'crt1'
   cflags = ['-O2']
   src_dir = ['system', 'lib', 'libc']
-  src_files = ['crt1.c']
+  src_files = ['crt1.c', '__original_main.c']
 
   def get_ext(self):
-    return '.o'
+    return '.a'
```

Comment:
I would put __original_main.c in libc and keep crt1.o as an object.  That is the traditional linker model.

Reply:
Hmm, but this original_main stuff is specific to standalone mode, so I don't think libc is the right place? We'd need to vary libc on standalone mode in that case.

Reply:
As long as it not pulled in there won't be any issue its just an unused object in the .a file.

Reply:
Interesting, I tested that, but it looks like that won't work with main modules where we link in the entire libc? We'd force that file to be included in that case, which is not good - it may even be a duplicate symbol if the program has a main without argv.

Reply:
I can rename the `crt1` library to something more general, that contains `crt1.o` and `__original_main.o`, perhaps? Or maybe they could both be in `libstandalone-wasm`?